tools/datasetSnapshotHelperScripts/files/OMOPDataset/populate_omop_data.py

```diff
@@ -0,0 +1,152 @@
+###
+# This script is used to generate JSON files with tabular OMOP data with a small test subset of OMOP data
+# It queries the BigQuery dataset and writes the results to json files to be used in the setup_ui_integration.py script
+# (1) We start with the person table and pull a small subset of person ids
+# (2) Then we retrieve entries the occurrence tables (procedure_occurrence, condition_occurrence, drug_exposure) that map to the selected person ids,
+#       noting the referenced concept_ids in these entries
+# (3) We pull all the records from some of the reference tables (right now, just the domain table, but we could include others)
+# (4) We pull entries from the concept table that match referenced concept_ids from the occurrence tables
+# (5) We populate the concept_ancestor table with the ancestor_concept_id pulled from occurrence table's domain id and
+#       then the descendant_concept_id contains all the concepts form the concept table.
+#       This is the only table that we are fabricating.
+# (6) We write the results to json files
+# (7) We use the json files in the setup_ui_integration.py script to populate a test environment with the OMOP data
+#
+# How to use this script
+# Note: this script overwrites the json files each time it is run
+# (1) Set the gcp_with_access variable to a google project that you have job big query access on
+# (2) Set the gcp_project and gcp_dataset variables to the project and dataset that you want to pull data from
+# (3) Set the person_limit variable to the number of person records you want to pull.
+#       Note: Increasing this will very quickly increase the size of the tables, especially the concept table.
+
+# Running the script
+#1. If you need to set up a virtual python environment, you can run the following commands:
+#    * `python3 -m venv c:/path/to/myenv`
+#    * `source c:/path/to/myenv/bin/activate`
+#2. `cd jade-data-repo-ui/tools/ui_integration`
```

Comment:
Can you update the path here?
```suggestion
#2. `cd jade-data-repo/tools/datasetSnapshotHelperScripts`
```